scripts/import_assets/import_assets.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import json
+import requests
+import argparse
+import os
+from requests.exceptions import HTTPError
+from urllib.parse import urljoin
+
+DEFAULT_API_URL = 'https://se-api.holmsecurity.com/v1/'
+
+
+def read_data(args):
+    # reads data from a csv_file
+
+    with open(args.csv_path, 'r') as csv_file:
+        csv_reader = csv.reader(csv_file)
+        for row in csv_reader:
+            ip = row[5]
+            asset_type = get_asset_type(ip)
+            name = row[0]
+            business_impact = row[1]
+            details = row[2]
+            hosts_personal_data = row[4]
+            tags = [t for t in row[3].split('|') if t]
+            dict_fields = {"name": name,
```

Comment:
Please use `yapf -i` on the file to format the code style.

Reply:
Done!